rider-core/src/main/java/com/github/database/rider/core/dataset/DataSetExecutorImpl.java

```diff
@@ -901,6 +902,22 @@ public void setDBUnitConfig(DBUnitConfig dbUnitConfig) {
             this.dbUnitConfig = dbUnitConfig;
             riderDataSource = null;
             tableNameResolver = new TableNameResolver(dbUnitConfig);
+            setDefaultDBUnitEscapePattern();
+        }
+    }
+
+    private void setDefaultDBUnitEscapePattern() {
+        Object configEscapePattern = dbUnitConfig.getProperties().get("escapePattern");
+        if (connectionHolder != null && (configEscapePattern == null || "".equals(configEscapePattern))) {
+            try {
+                Connection connection = connectionHolder.getConnection();
+                if (connection != null) {
+                    String escapePattern = connection.getMetaData().getIdentifierQuoteString();
+                    dbUnitConfig.addDBUnitProperty("escapePattern", escapePattern);
+                }
+            } catch (SQLException e) {
+                log.warn("Could not set default DBUnit escape pattern: " + e.getMessage(), e);
+            }
```

Comment:
I would not try to set a default escaping, the escapePattern from dbunitConfig should be used only when user provides it otherwise we'll use the one from tableMetadata. I would remove this method, WDYT?

Reply:
hi @rmpestano, I just found that I need to set the default DBUnit `escapePattern` when I tried to run database testing on my project. It will delete all data on the table after each test, and it was executed by DBUnit `DatabaseOperation.CLEAN_INSERT` as defined in the `DataSetConfig` default strategy. The changes on `TableNameResolver` were not helping on that part, because the query was done by DBUnit. 

So I tried to set that default `escapePattern` to fix my database tests, I didn't need to create different `dbunit.yml` for every database (didn't find a way to set it from the command line) and I thought it would be useful for me as a user of DBRider don't need to know the `escapePattern` value I need to set. Please let me know what you think.

Reply:
> That operation threw an error due to unescaped table name if I didn't set the escapePattern value

I just removed this method (and its references) and all tests passed, can you try that?

> and I thought it would be useful for me as a user of DBRider don't need to know the escapePattern value I need to set

We can mention on docs that if user wants to override dbunit escapePattern they need to set the `escapePattern` property in dbrider (yml or `@DBUnit`) otherwise we'll use the pattern provided by dbunit. WDYT?

Reply:
Well noted. I just updated the code, please kindly check again.

Reply:
Looks great, will just wait the build to pass and merge it, thank you again!

Reply:
It's my pleasure to work on your cool project. Thank you!

Reply:
The build still fails, it was all passed on my last test.

<img width="575" alt="Jepretan Layar 2021-08-09 pukul 21 58 13" src="https://user-images.githubusercontent.com/11657869/128728389-25812a59-3954-489b-a424-ad852e35fb4b.png">

Reply:
It's related to github ci, the tests are passing